File: java/src/main/java/model/params/TradeParams.java
package model.params;

import model.ChannelTypes;

public class TradeParams extends Params {

    public TradeParams(String symbol) {
        super(ChannelTypes.TRADE, symbol);
    }

    @Override
    public String toString() {
        return "TickerParams{" +
                super.toString() +
                "} ";
    }
}

File: c-sharp/ws-client/model/parameters/TradeParams.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ws_client.model.parameters
{
    class TradeParams : Params
    {
        public TradeParams(string symbol) : base(ChannelTypes.TRADE, symbol)
        {
        }
    }
}
